app/services/EmailService.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import com.google.inject.Inject
+import connectors.EmailConnector
+import models.email.requests.{FailedXmlSubmissionPlatformOperator, FailedXmlSubmissionUser, SendEmailRequest, SuccessfulXmlSubmissionPlatformOperator, SuccessfulXmlSubmissionUser}
+import models.operator.responses.PlatformOperator
+import models.submission.Submission.State.{Approved, Rejected}
+import models.subscription.responses.SubscriptionInfo
+import org.apache.pekko.Done
+import uk.gov.hmrc.http.HeaderCarrier
+
+import scala.concurrent.Future
+
+class EmailService @Inject()(emailConnector: EmailConnector) {
+
+  def sendSuccessfulBusinessRulesChecksEmails(state: Approved, checksCompletedDateTime: String, platformOperator: PlatformOperator, subscriptionInfo: SubscriptionInfo)
+                                   (implicit hc: HeaderCarrier): Future[Done] = {
+    sendEmail(SuccessfulXmlSubmissionUser.build(state, checksCompletedDateTime, platformOperator, subscriptionInfo))
+    sendEmail(SuccessfulXmlSubmissionPlatformOperator.build(state, checksCompletedDateTime, platformOperator))
```

Comment:
This looks like we're sending emails to both the user and the PO, even when they are both the same (so the same person will receive both emails).  Is this what we want?

Reply:
There is a separate ticket ODPR-1707 for duplicate emails.

Reply:
Shouldn't we just do it right in the first place though?  I understand we need to go back and avoid duplicates on the emails we already have, but this PR is to add email functionality here; I think we should just do it properly rather than knowingly doing it wrong and planning to come back later.